azurerm/data_source_firewall.go

```diff
@@ -32,10 +32,9 @@ func dataSourceArmFirewall() *schema.Resource {
 
 			"resource_group_name": azure.SchemaResourceGroupNameForDataSource(),
 
-			"ip_configuration": {
+			"ip_configurations": {
```

Comment:
Do you think we should change the name? Wouldn't this break everyone that is already using this resource? I think just removing the `MaxItems` should be enough here, WDYT?

Reply:
Agree and reverted.